buildout.cfg

```diff
@@ -1,14 +1,20 @@
 [buildout]
-extends = http://dist.plone.org/release/4-latest/versions.cfg
+extends = http://dist.plone.org/release/5.2-latest/versions.cfg
```

Comment:
I usually try not to rely on "unstable" extends like "latest". This is subject to change and can break the CI build "from the outside". So I'd rather rely on 5.2.1 and upgrade manually.